packages/appium/lib/main.js

```diff
@@ -216,18 +216,18 @@ async function init (args) {
   // 2. config file
   // 3. defaults from config file.
   if (parsedArgs.subcommand === SERVER_SUBCOMMAND) {
+    const flatDefaults = getDefaultsFromSchema();
+    const defaults = {};
+    for (const [key, value] of Object.entries(flatDefaults)) {
```

Comment:
can we use Object.assign?

Reply:
I'm not sure how.  Some keys in this object are of the format `extType.extName.argName` which would get ugly.